docs/resources/itsi_kpi_threshold_template.md

```diff
@@ -3,14 +3,123 @@
 page_title: "itsi_kpi_threshold_template Resource - terraform-provider-splunk-itsi"
 subcategory: ""
 description: |-
-  
+  Manages a KPI threshold template.
 ---
 
 # itsi_kpi_threshold_template (Resource)
 
-
-
-
+Manages a KPI threshold template.
+
+## Example Usage
+
+```terraform
+resource "itsi_kpi_threshold_template" "example" {
```

Comment:
nothing required to be changed but here could be mentioned that _time-variate static_ threshold could be supported only via template like:
policy_type = "static"
and then dynamic_param is force set to 0

Reply:
Thanks! I added an example for a static threshold template.